tests/model_options/test_default_related_name.py

```diff
@@ -25,6 +29,25 @@ def test_default_related_name(self):
         except AttributeError:
             self.fail("Author should have a books relation.")
 
+    def test_default_related_name_in_queryset_lookup(self):
+        try:
```

Comment:
Please revise the try/except pattern in light of 7fec264e46d2a757f06f857e513072d72686cf9d.